tests/test_storage_tools.py

```diff
@@ -9,16 +9,16 @@
 from keboola_mcp_server.sql_tools import TableFqn, WorkspaceManager
 from keboola_mcp_server.storage_tools import (
     BucketDetail,
+    TableColumnInfo,
```

Comment:
It seems I have different isort settings because I get case insensitive sorting :/